src/Sarif/ExtensionMethods.cs

```diff
@@ -336,5 +344,105 @@ public static string GetFirstSentence(string text)
             }
             return text.TrimEnd('.') + ".";
         }
+
+        /// <summary>Retrieves a property value if it exists, or null.</summary>
```

Comment:
>Retrieves [](start = 21, length = 9)

Dude! You have repeatedly busted my chops about doc comments ("Don't use"). I'm happy to have you use them. Please don't change this.